app/components/resend-invitation/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: '', // tagless component
+
+  store: Ember.inject.service(),
```

Comment:
ah, good point. It probably makes sense to remove the global setup-component-with-store initializer in favor of targeted injections a la `Ember.service.inject`, though, yeah?